languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,19 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="IL_FAUT" name="Il faut" default="temp_off">
```

Comment:
Judging from the rule editor results, this rule will have a pretty high impact on users. We should make it more precise.

For now, it doesn't care whether the sentence is "Faut que j'y aille" or "Faut pas l'oublier"

We should have a specific subrule for "Faut" being followed by pas|plus|... that suggests the formally right way of phrasing "Il ne faut pas".